.pre-commit-config.yaml

```diff
@@ -10,22 +10,24 @@ repos:
   hooks:
   - id: isort
 - repo: https://github.com/psf/black
-  rev: 23.3.0
+  rev: 23.7.0
   hooks:
   - id: black
-    language_version: python3
+    args: [--target-version=py37]
 - repo: https://github.com/pre-commit/mirrors-mypy
   rev: 'v1.4.1'
   hooks:
   - id: mypy
-    additional_dependencies: ["numpy"]
-    exclude: ^examples/
+    args: [--python-version=3.7]
```

Comment:
Please add the python-version config for mypy via pyproject.toml

Reply:
Done